runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargeConfig.java

```diff
@@ -203,4 +203,48 @@ default boolean recoilNotification()
 	{
 		return false;
 	}
+
+	@ConfigItem(
```

Comment:
This should just respect item charges config, and should be just boolean displayInfoboxes